build/podman/src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+quarkus.container-image.group=qe
+# DON'T remove the spaces that are at the end of quarkus app name
+quarkus.application.name=hello-world-app      
```

Comment:
I'd prefer to call it differently than in the Docker module if you don't mind. Please add `-podman` postfix or something, especially as the image is deleted in `@AfterAll` not before the tests. If all conditions play right (like `@Tag("podman-incompatible")`, both modules shouldn't be run together, but both modules are still built, right?

BTW what happens when you build `docker-build` module with Podman, did you test it?